compute/cloud-client/src/main/java/compute/DeleteInstance.java

```diff
@@ -19,10 +19,13 @@
 // [START compute_instances_delete]
 
 import com.google.cloud.compute.v1.InstancesClient;
+// [START compute_instances_operation_check]
```

Comment:
Do we need to include the imports separately? Ideally this should be part of the whole snippet: go/cloud-sql-dpe-terraform

Reply:
In order to provide more context (on import packages) to the customers using the sample, this approach has been adopted, across languages for this particular region tag.
Please refer to: https://github.com/GoogleCloudPlatform/php-docs-samples/blob/master/compute/cloud-client/instances/src/delete_instance.php

Making the "compute_instances_operation_check" as a standalone snippet proved to be complex to demonstrate and wasn't intuitive, hence went with the above approach.

Reply:
Do you have a one pager or write up somewhere describing why this is necessary? This is contradictory to go/code-snippets-style, which says "A code snippet demonstrates one task and only one task."

Reply:
The operation_check snippet is not a standalone snippet and would always have to be used with another API, say create or delete.
If we make this "operation check" an independent snippet, we'll anyway have to reference it from other snippets, which would inturn be a contradiction to the snippets-style.

I can think of a solution to this:
Could we conform to the snippets-style by increasing a little bit of redundancy? 
1. We can keep the "operation check" in the create and delete snippets as such, but remove the region tags.
2. Create an new snippet for "operation check" and that can have the region tags.

wdyt?

Reply:
That SGTM. Feel free to ping me when you need me to review.